bigbluebutton-client/src/org/bigbluebutton/modules/whiteboard/views/WhiteboardCanvas.as

```diff
@@ -275,6 +284,11 @@ package org.bigbluebutton.modules.whiteboard.views {
 			addChildAt(newGraphicHolder, getChildIndex(graphicObjectHolder));
 			removeChild(graphicObjectHolder);
 			graphicObjectHolder = newGraphicHolder;
+			
+			var children:IChildList = this.graphicObjectHolder.rawChildren;
+			while (children.numChildren != 0) {

```

Comment:
The old 2.2 code was clearing the annotations by creating a new graphicObjectHolder and letting the old one be garbage collected. The the 2.0 green code was clearing the annotations by removing each of the children in the graphic holder. We don't need both approaches included in the function.